files/master.cfg

```diff
@@ -31,6 +30,18 @@ class WorkerAdmins:
         except KeyError:
             return None
 
+{% endraw %}
+class Settings:
+    pass
+settings = Settings()
+settings.title = '{{ afsbotcfg_title }}'                 # Title for buildbot banner
+settings.summary_title = '{{ afsbotcfg_summary_title }}' # Title used in gerrit summary
+settings.repo = '{{ afsbotcfg_repo }}'                   # Gerrit repo
+settings.buildbotURL = '{{ afsbotcfg_url }}'             # URL to buildbot
+settings.www_port = {{ afsbotcfg_www_port }}             # Buildbot port
+settings.build_delay = {{ afsbotcfg_build_delay }}       # Time to delay start of builds
```

Comment:
This looks good, thanks. I think this would work too:

    class Settings:
         title = '{{ .... }}'
         repo = '{{ ... }}'
         ...

    settings = Settings()

Reply:
Yes -- I just used the pattern from the python doc on namespaces..